server/client_manager.py

```diff
@@ -864,6 +864,13 @@ def change_area(self, area, password=""):
             ):
                 if not old_area.dark and not old_area.force_sneak:
                     if old_area.area_manager == self.area.area_manager:
+                        if self.area.area_manager.passing_msg == True:
+                            if showname == '':
+                                showname = self.char_name
+                            old_area.send_ic(
+                                None, '1', 1, "", "", f'~~[º{self.showname}º leaves to º{area.name}º.]', 
```

Comment:
Miscellaneous thing you can do, but to make these msgs as unobtrusive as possible, you can preface them with 
`f'~~}}}[`
so they type out immediately, or
`f'~~}}[`
so the text types out really fast.